### DISCUSSION TITLE
Pentingnya OrganizationID pada SATUSEHAT

### DISCUSSION THREAD

**Discussion by hitaaksataDev8** [ivanwilliammd/satusehat-integration#120] [Q&A (Tanya Jawab)] [answered]
halooo
mohon bantuan untuk pengiriman data ke encounter

saya mendapatkan respon
`{
    "resourceType": "OperationOutcome",
    "text": {
        "status": "generated"
    },
    "issue": [
        {
            "severity": "error",
            "code": "value",
            "details": {
                "text": "Wrong organization ID provided by system URL: http://sys-ids.kemkes.go.id/encounter/100025753 (RuleNumber: 10117)"
            },
            "expression": [
                "Encounter.identifier[0].system"
            ]
        }
    ]
}`

saya sudah menggunakan organization id dari master sarana
playload :
`{
    "resourceType": "Encounter",
    "identifier": [
        {
            "system": "http://sys-ids.kemkes.go.id/encounter/100025753",
            "value": "20240221000060"
        }
    ],
    "status": "arrived",
    "period": {
        "start": "2024-03-18T04:54:02+00:00"
    },
    "statusHistory": [
        {
            "status": "arrived",
            "period": {
                "start": "2024-03-18T04:54:02+00:00"
            }
        }
    ],
    "class": {
        "code": "AMB",
        "display": "ambulatory",
        "system": "http://terminology.hl7.org/CodeSystem/v3-ActCode"
    },
    "subject": {
        "reference": "Patient/P01934888239",
        "display": "LUSIA LERO"
    },
    "participant": [
        {
            "individual": {
                "reference": "Practitioner/1000687602",
                "display": "dr. ADE SINYO ARISTANTRISNA, Sp.B"
            },
            "type": [
                {
                    "coding": [
                        {
                            "system": "http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
                            "code": "ATND",
                            "display": "attender"
                        }
                    ]
                }
            ]
        }
    ],
    "location": [
        {
            "location": {
                "reference": "Location/7910d7a1-0383-4858-92c9-64471a32c587",
                "display": "Klinik Bedah"
            }
        }
    ],
    "serviceProvider": {
        "reference": "Organization/100025753"
    }
}
`

apakah ada yang salah dengan data yang saya kirim
mohon bantuannya

**Comment by odilianafirishavon** [comment]
sudah prod ya? 
coba dev/stg, response nya masih sama ga?

**Comment by ivanwilliammd** [comment, marked as answer]
Untuk kendala ini biasanya disebabkan karena clientID & clientSecret-nya tidak sesuai . Silakan cek organizationID dan replace 100025753 dengan organizationID yang sesuai @hitaaksataDev8 

environment dev, staging, prod memiliki organization ID yang berbeda
100025753 --> bentuk digit angka ini biasanya adalah organizationID production

**Comment by hitaaksataDev8** [comment]
halo sudah bisa skrg, setelah saya ganti organization ID nya yang menggunakan UUID bukan yang dari sarana index

terima kasih
